keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/unary/SumVertexTest.java

```diff
@@ -122,4 +125,15 @@ public void canDoSumAutoDiffWhenWrtIsScalar() {
         assertThat(dfdaForward, equalTo(expectedDfdx));
         assertThat(dfdaReverse, equalTo(expectedDfdx));
     }
+
+    //@Test - XXX - The sum vertex produces dual numbers with "weird" shape.  Discuss.
```

Comment:
You can use the `@Ignore("reason")` annotation

Reply:
Thanks - that's Perfect